packages/api-explorer/src/components/AuthForm/index.jsx

```diff
@@ -0,0 +1,136 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import {Divider} from 'antd'
+import debounce from 'lodash.debounce'
+import pick from 'lodash.pick'
+import {omit} from 'ramda'
+
+const SecurityInput = require('../../SecurityInput')
+
+const styles = {
+  container: {
+    display: 'grid',
+    gridTemplateRows: 'repeat(auto-fit, minmax(50px, 1fr))'
+  },
+  inputsContainer: {
+    paddingTop: 10
+  },
+  schemeName: {
+    fontSize: 16
+  }
+}
+
+function omitOrMerge (current, value, key) {
+  if (!value) {
+    return omit([key], current)
+  }
+
+  return {
+    ...current,
+    [key]: value
+  }
+}
+
+function getSecuritySections(securityTypes, config, onChange, onSubmit, schemeName) {
+  const {authInputRef, oauth, auth} = config
+  return (
+    <form onSubmit={onSubmit}>
```

Comment:
This is a form with a submit without a submit button?

Reply:
Actually yes, because the form data are kept until you press the Try It button which resides in another component